src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольный треугольник
+    private double height; // высота
+
+    public RightTriangle(String color, double firsSide, double height) {
+        super(color, firsSide, height);
+    }
+
+    @Override
+    public double detAreaOfFigure() {
+        return (getFirsSide() * height) / 2;
```

Comment:
There is a typo in the method name `getFirsSide`. It should be `getFirstSide` to match the corrected variable name in the `FigureTwo` class.